build.gradle

```diff
@@ -237,8 +237,8 @@ dependencies {
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
-  compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '3.0.4'
-  compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: '5.1.1'
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '5.0.1'
```

Comment:
hold this off as tests need to be updated which rely on sync logging. after that is solved - can even move to very latest version

Reply:
reverted the changes